api-tests/apply_route_test.go

```diff
@@ -114,12 +114,12 @@ func TestApplyPgShouldReturnSameMajorVersion(t *testing.T) {
 
 	pgParams := &version_service.VersionServiceApplyParams{
 		Apply:           "latest",
-		OperatorVersion: "1.0.0",
+		OperatorVersion: "1.1.0",
 		Product:         "pg-operator",
 	}
 	pgParams.WithTimeout(2 * time.Second)
 
-	for _, v := range []string{"12.8", "13.4"} {
+	for _, v := range []string{"12.8", "13.4", "14.1"} {
```

Comment:
```suggestion
	for _, v := range []string{"12.8", "13.5", "14.1"} {
```
in 1.1.0 we will have PG 12.8 , 13.5 and 14.1 please fix PR according this information

Reply:
done